src/components/sideBar/index.js

```diff
@@ -111,7 +122,15 @@ const SideBar = () => {
                   body='Are you sure want to logout?'
                   buttonLeftText='Cancel'
                   buttonRightText='Yes'
+<<<<<<< HEAD
+                  popupOpen={logoutPopup}
+                  setPopupOpen={setLogoutPopup}
+                  onSubmit={onSubmit}
+                  buttonRightColor='red'>
+                </AlertComponent>
+=======
                   buttonRightColor='red' />
+>>>>>>> 6e97c8854787027d674b048ba41567f9b63488a6
```

Comment:
ada conflict belum ke fix ni wkwkk 😄

Reply:
HAHA iya tdi baru sadar pas gagal build 😂 thank you wkwk